app/scripts/lib/transaction/metrics.ts

```diff
@@ -800,6 +800,14 @@ async function buildEventFragmentProperties({
     transactionMetricsRequest.getTokenStandardAndDetails,
   );
 
+  let contractMethodName;
+  if (transactionMeta.txParams.data) {
```

Comment:
Minor, could we assign the data to a variable to avoid the duplicate property chain?